app/assets/javascripts/application.js.erb

```diff
@@ -457,15 +472,17 @@ $(document).ready(function() {
     }
 
     var replies = comment.nextAll(".comments").first();
-    $.get("/comments/" + comment.attr("data-shortid") + "/reply",
-    function(data) {
-      var reply = $($.parseHTML(data));
-      reply.attr("id", "reply_form_" + comment.attr("id"));
-      replies.prepend(reply);
-      var ta = reply.find("textarea");
-      ta.focus().text(sel);
-      autosize(ta);
-    });
+    var done = Lobsters.loading($(this), () => {
```

Comment:
The arrow function here and on line 251 are our first usage. So my old fart question is how universal browser support is for them now? We don't have any preprocessing step to transpile down, and I really don't want one.